src/components/PeriodicReports/PeriodicReportsList.tsx

```diff
@@ -26,12 +26,14 @@ export const PeriodicReportsList = ({
   breadcrumbs = [],
   pageTitleSuffix,
   reports,
+  requirementCheckList,
   onRowClick,
 }: {
   type: ReportType;
   breadcrumbs?: ReactNode[];
   pageTitleSuffix?: string;
   reports?: readonly PeriodicReportFragment[];
+  requirementCheckList?: ReactNode;
```

Comment:
Maybe this could just be `children` instead? I think that would be a bit more idiomatic for React.
As in the caller can override the children or we'll show the table by default.